src/Json.php

```diff
@@ -37,9 +37,9 @@ public static function decode(string $string)
     /**
      * @param mixed $content
      */
-    public static function encode($content): string
+    public static function encode($content, int $options = 0): string
```

Comment:
while at it, should it also add the 3rd param ?

Reply:
Right, using enums/helpers to define options could be an idea.

TBH I chose this package because I wanted a safer json encode/decode (exceptions) that would be the easiest to replace existing calls. If I had to learn a new custom API I wouldn't probably have used it.

> Type safe json encode/decoder, the goal is to not leave errors unchecked.

(from the README)

This is what got me interested, and I chose it because it was the most straightforward and lightest option. I found many projects adding "opinionated" behavior on top of json decode/encode and that wasn't really interesting.

I also considered using [safe](https://github.com/thecodingmachine/safe) but for Bref/AWS lambda I'm worried of the performance impact (1ms is important given the existing cost of PHP on lambda).

Reply:
Totally agree with @mnapoli